src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -157,6 +166,42 @@ public ServerMock()
 		logger.setLevel(Level.ALL);
 	}
 
+	@NotNull
+	public ServerSettingsMock getServerSettings(){
+		if (serverSettings == null)
+		{
+			serverSettings = new ServerSettingsMock();
+			try
+			{
+				File file;
+				URL systemResource = ClassLoader.getSystemResource(BUKKIT_SETTING_FILE);
+				if (systemResource != null)
+				{
+					file = new File(systemResource.toURI());
+				}
+				else
+				{
+					file = new File(BUKKIT_SETTING_FILE);
+				}
+
+				if (!file.exists())
+				{
+					Path tempFile = Files.createTempFile(file.getName(), String.valueOf(Math.random()));
+					file = new File(tempFile.toUri());
+					file.deleteOnExit();
```

Comment:
Could we just use an in-memory configuration instead of creating a new file every time?